### PR TITLE
Localize commands

### PR DESCRIPTION
**Pull request by Ygg01**
A continuation of #2331 and stepping stone for #2273.

Just without too much complication. Rather than a custom data structure, adds an abstract `LocalizeCommand` that preserves `IConsoleCommand` interface.
`LocalizeCommand` standardizes the `IConsoleCommand` localization to `cmd-{Command}-{help/desc}` for `Help` and `Description` respectively.

Related to https://github.com/space-wizards/space-station-14/issues/5543 (Expect a commit there too as well).

Non-breakable change at first glance.

### CODE REVIEW

**Comment by Ygg01** [comment]
Hrm, why is entityQuery failing on transform in Content.IntegrationTests?

NVM. It was some intermittent failure.

File: Resources/Locale/en-US/commands.ftl [hunk 1]
```diff
@@ -159,3 +159,339 @@ cmd-hint-savebp-id = <Grid EntityID>
...
+cmd-ldrsc-help = Usage: ldrsc <path> <type>
+
+cmd-rldrsc-desc = Reloads a resource.
+cmd-rldrsc-help = Usage: rldrsc <path> <type>
+
+cmd-gridtc-desc = Gets the tile count of a grid.
+cmd-gridtc-help = Usage: gridtc <gridId>
+
+
+# Client site commands
```
Comment (PJB3005) [inline Resources/Locale/en-US/commands.ftl]:
```suggestion
# Client-side commands
```

File: Robust.Shared/Console/LocalizedCommands.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    [Dependency] private readonly ILocalizationManager _loc = default!;
+
+    /// <inheritdoc />
+    public abstract string Command { get; }
+
+    /// <inheritdoc />
+    public string Description => _loc.TryGetString($"cmd-{Command}-desc", out var val) ? val : "";
+
+    /// <inheritdoc />
+    public string Help => _loc.TryGetString($"cmd-{Command}-help", out var val) ? val : "";
```
Comment (PJB3005) [inline Robust.Shared/Console/LocalizedCommands.cs]: These should stay virtual IMO. Some commands dynamically build the help body up from enum members etc.

Comment (Ygg01) [reply Robust.Shared/Console/LocalizedCommands.cs]: Thanks, that solved some problems with VV Commands.

Here is when I point SS14/RobustToolbox to this branch.
Content Client_pa77AcGAa6

**Review by ElectroJr — CHANGES_REQUESTED** [review]
This currently breaks completion hints (e.g., see `/help`'s lack of completions)

**Pull request merged**